### ISSUE TITLE
Accessibility issue: links "click here"

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/openstreetmap-website#5431]
Copy from [Translatewiki](https://translatewiki.net/w/i.php?title=Translating_talk:OpenStreetMap&oldid=12791558#Accessibility_issue:_Osm:Confirmations.confirm.click_here/en,_Osm:Confirmations.confirm.resend_html/en):

The current message text is:

* **Confirmations.confirm.resend html**: <code>If you need us to resend the confirmation email, %{reconfirm_link}.</code>
* **Confirmations.confirm.click here**: <code>click here</code>

For accessibility, “click here” should never be a link text, because the link text should always describe the link. A more accessible version of these messages would be:

* **Confirmations.confirm.resend html**: <code>If needed, we can %{reconfirm_link} to you.</code>
* **Confirmations.confirm.click here**: <code>resend the confirmation email</code>

[Rationale from the W3C](https://www.w3.org/TR/WCAG20-TECHS/F84.html): _Many blind people who use screen readers call up a dialog box that has a list of links from the page. They use this list of links to decide where they will go. But if many of the links in that list simply say "click here" or "more" they will be unable to use this feature in their screen reader, which is a core navigation strategy. That's why it's a failure of 2.4.9 to not provide any way of allowing them to know the destination from the link text alone. It is also true for people who tab through links. If all they hear as they tab through the document is "click here, click here, click here etc." they will become confused._

**Comment by peach280** [comment]
I have generated a PR to fix this issue. Please review and provide your valuable feedback

**Comment by tomhughes** [comment]
In reality nobody should ever see that version of the page - the only way that is ever rendered is if you somehow arrive at the confirm link without a token parameter in the URL which should never happen.

**Comment by AntonKhorev** [comment]
> I have generated

Generated with what?

**Comment by AntonKhorev** [comment]
> In reality nobody should ever see that version of the page - the only way that is ever rendered is if you somehow arrive at the confirm link without a token parameter in the URL which should never happen.

It happens after you fill out the signup from and click "Sign Up". We should probably remove redirects/renders from `save_new_user` in users controller to make it more clear.
